atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -93,7 +93,7 @@ protected int getPoolSize() {
             return f.apply(resource);
         } catch (Exception e) {
             if (isInvalidClientConnection(e)) {
-                log.warn("Not reusing resource {} due to {}", resource, e);
+                log.warn("Not reusing resource {} due to {}", resource, e.toString());
```

Comment:
Are we happy with this no longer printing the stack?  toString (generally) will only return the exception name : Message of Exception as compared to the log method which will return the whole trace.